### ISSUE TITLE
Foundry branch still imports hardhat

### ISSUE THREAD

**Issue by didier-moet** [scaffold-eth/scaffold-eth-2#681]
packages/nextjs/components/Footer.tsx
Line 01
Footer contains references to hardhat when it should be Foundry

**Comment by technophile-04** [comment]
Hey thanks @didier-moet, it seems that in Footer we are just referencing harhdat for its chainId so it shouldn't cause any issue with functionality, so simplicity sake we have kept it as is but in future hopefully it will be removed   

Are facing any issue or noticed any weird because of this in functionality ?

**Comment by KcPele** [comment]
Should I work on this or let it be for now?

**Comment by technophile-04** [comment]
I think its fine for now for simplicity sake, since it does not have any side effects like showing `hardhat` instead of `foundry` on UI etc, thanks @didier-moet for raising this, please feel free re-open / comment if there's some edge case / bug which we are missing 🙌

**Comment by didier-moet** [comment]
Well, there might be an edge case.
I'm running Scaffold Eth on a VPS, so everywhere 'localhost' or '127.0.0.1' is mentioned, I had to change it to point to my server.
Initially I just dug into the packages/nextjs/node_modules/viem/chains/definitions/foundry.ts file, modified the rpcUrls to point to my server. (Just to see if it would work)
Didn't work for some reason.
So I just modified the 'chains' object within 'scaffold.config.ts' so that the rpcUrls point to my server. This worked, though I still got some "Cannot connect to local provider" errors.
This set me on a hunt for wherever 'localhost' might still be referenced

Still getting the errors, but apparently it's connecting to my server because I'm able to use the debug interface and the faucet but not the block explorer

My thought is that I haven't found all the references to 'localhost'.
Your thoughts are welcome
